packages/block-library/src/navigation/edit/menu-inspector-controls.js

```diff
@@ -70,6 +72,8 @@ const MenuInspectorControls = ( {
 		? 'list'
 		: undefined;
 
+	const hasClassicOrNavigationMenus = useHasClassicOrNavigationMenus();
+
```

Comment:
Given the UI is not the more menu and one button in the tools section, that button can be there all the time. So we don't need to check for this anymore.